include/picongpu/particles/ionization/byField/BSI/BSI_Impl.hpp

```diff
@@ -26,7 +26,7 @@
 #include "picongpu/fields/FieldE.hpp"
 #include "picongpu/fields/FieldJ.hpp"
 #include "picongpu/particles/ParticlesFunctors.hpp"
-#include "picongpu/particles/atomicPhysics/SetToAtomicGroundStateForChargeState.hpp"
+#include "picongpu/particles/atomicPhysics/SetChargeState.hpp"
```

Comment:
please us ethe include in all param files too or remove them if `SetToAtomicGroundStateForChargeState` is not used there.
in test and there the collision test `SetToAtomicGroundStateForChargeState` is used and must be substituted by `SetChargeState` too.